src/OrchardCore.Modules/OrchardCore.Lists/GraphQL/ListQueryObjectType.cs

```diff
@@ -36,9 +37,9 @@ public ListQueryObjectType(IStringLocalizer<ListQueryObjectType> S)
 
                     var dataLoader = accessor.Context.GetOrAddCollectionBatchLoader<string, ContentItem>("ContainedPublishedContentItems", x => LoadPublishedContentItemsForListAsync(x, session));
 
-                    return (await dataLoader.LoadAsync(g.Source.ContentItem.ContentItemId))
+                    return ((await dataLoader.LoadAsync(g.Source.ContentItem.ContentItemId).GetResultAsync())
```

Comment:
i need to read up on the dataloader changes; but this doesnt seem right ... seems like it would return the results immediately rather then batching them into a dataloader